libnetwork/libnetwork_linux_test.go

```diff
@@ -45,7 +45,7 @@ const (
 
 func TestMain(m *testing.M) {
 	// Cleanup local datastore file
-	_ = os.Remove(datastore.DefaultScope("").Path)
+	_ = os.Remove(datastore.NewConfig("").Path)
```

Comment:
See my other comment; looks like if we export `DefaultPath`, then this code wouldn't have to call `NewConfig` and could just remove the `DefaultPath` (which would be `/var/lib/docker/network/files/local-kv.db`)